cmd/logout.go

```diff
@@ -0,0 +1,26 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var componentLogoutCmd = &cobra.Command{
+	Use:   "logout",
+	Short: "Log out of the active session out by clearing saved tokens",
```

Comment:
Log out of the active session?

I think we don't need to say clearing saved tokens to the user ?